ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-upgrades.yaml

```diff
@@ -0,0 +1,30 @@
+releases:
```

Comment:
Why did you need this upgrade variant config?

Reply:
Not sure if the difference between `nightly` and `ci` really matters much for the periodic upgrade tests though.

Reply:
We do want them to be identical

Reply:
I guess we will want a set of `ocp-4.5-ci` and `ocp-4.5-nightly` and `ocp-4.5-prerelease` variants (defined by what it means to be `latest` in each config)

Reply:
Renamed the new variant to `ocp-4.5-ci`